frontend/packages/dev-console/integration-tests/support/step-definitions/helm/helm-release.ts

```diff
@@ -97,9 +97,37 @@ When('user clicks on the Uninstall button', () => {
 });
 
 Then('user will be redirected to Topology page with no workloads', () => {
-  cy.document()
-    .its('readyState')
-    .should('eq', 'complete');
```

Comment:
Does this lines creating issue?

Reply:
it was not required here and hence removed. it was not creating any issue.